WordPress/src/main/java/org/wordpress/android/ui/photopicker/PhotoPickerFragment.kt

```diff
@@ -17,6 +17,7 @@ import androidx.lifecycle.viewModelScope
 import androidx.recyclerview.widget.GridLayoutManager
 import com.google.android.material.dialog.MaterialAlertDialogBuilder
 import org.wordpress.android.R
+import org.wordpress.android.R.string
```

Comment:
Minor (🔍): As per my previous [such comment](https://github.com/wordpress-mobile/WordPress-Android/pull/17393#discussion_r1010454204) please consider removing this extra import and revert it back to how it was.

PS: When you commit a change please be careful on other changes, like import updates and revert those before committing.

Reply:
1. regarding the one commit to two refactor thing, i did that because i felt the second method is just few lines, will stick to using a commit per refactor regardless of the file changes moving forward.